SensioLabs/Security/SecurityChecker.php

```diff
@@ -17,9 +17,19 @@
 
 class SecurityChecker
 {
+    /**
+     * @var int
+     */
     private $vulnerabilityCount;
+    
+    /**
+     * @var CrawlerInterface
+     */
     private $crawler;
 
+    /**
+     * @param CrawlerInterface $crawler
```

Comment:
also, this phpdoc isn't meaningful, which was why it was omitted in the first place i assume